src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs

```diff
@@ -265,6 +265,15 @@ protected override async Task HandleChallengeAsync(AuthenticationProperties prop
             }
         }
 
+        protected override Task HandleForbiddenAsync(AuthenticationProperties properties)
+        {
+            var authenticationFailedContext = new AuthenticationFailedContext(Context, Scheme, Options)
+            {
+                Exception = new Exception("Forbidden")
```

Comment:
There's no reason to provide an exception here.

Reply:
removed this exception